### ISSUE TITLE
Add to Open VSX Registry

### ISSUE THREAD

**Issue by Invisible-Rabbit-Hunter** [Trebor-Huang/vscode-forester#10]
Would you consider adding this the the Open VSX Registry as well as the VSCode extension marketplace, so that people using e.g. VSCodium can install this extension easily as well?

**Comment by Trebor-Huang** [comment]
No, but I can license it so that you can upload this to the registry and maintain that yourself.
